src/utils/sort.js

```diff
@@ -1,12 +1,11 @@
-const normalizeBy = (type) => (value) => ({
+const normalizeBy = (type, value) => ({
```

Comment:
Please keep this the way before to support easy currying, and applying to things such as:

```js
const [aNorm, bNorm] = [a, b].map(normalizeBy(type))
```